### ISSUE TITLE
`in` keyword always tries to complete to Basics.isNan

### ISSUE THREAD

**Issue by brettcannon** [sbrink/vscode-elm#87]
When typing the `in` in a `let` expression, there's always an autocomplete suggestion for `Basics.isNan`. Not sure if this has something to do with the grammar file or how VS Code structures stuff that's causing a keyword to be autocompleted with a function.

**Comment by Krzysztof-Cieslak** [comment]
Yeah, it's known "feature" of very aggressive autocomplete in VSCode. In F# plugin I've end up with providing also list of keywords as part of results for autocomplete which solves this problem.

I guess we can do something like that here too.

**Comment by andys8** [comment]
I think the behaviour is hard to fix. One could argue that the prefix used for `isNan` could be changed to something different, e.g. `nan`. But even `identity` will autocomplete with `in`.

![image](https://user-images.githubusercontent.com/13085980/31152227-7f637e62-a89b-11e7-81d1-42ddb1e11ffb.png)

There is a bunch of vscode settings related to suggestions and there behaviour. Perhaps there is a way to improve it.

* `editor.snippetSuggestions`
* `editor.suggestOnTriggerCharacters`
* `editor.acceptSuggestionOnEnter`
* `editor.acceptSuggestionOnCommitCharacter`
* `editor.wordBasedSuggestions`

and [more](https://code.visualstudio.com/docs/getstarted/settings)
